adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryGtfsDataRepository.java

```diff
@@ -714,4 +714,14 @@ public StopTime addStopTime(final StopTime newStopTime) throws IllegalArgumentEx
     public Map<Integer, StopTime> getStopTimeByTripId(final String tripId) {
         return Collections.unmodifiableMap(stopTimePerTripIdStopSequence.get(tripId));
     }
+
+    /**
+     * Return an immutable map representing all records from stop_times.txt. Elements of this map are mapped by
```

Comment:
In the 2nd sentence, I would be clearer about the key and value definitions for both Maps, given it's a nested map. Something like "The key values for the returned map are X and the value is another map, which has a key of X and a value of Y and is ordered by ascending stop_sequence."